specification/datamigration/resource-manager/Microsoft.DataMigration/preview/2018-07-15-preview/definitions/Common.json

```diff
@@ -592,6 +592,20 @@
           "description": "Parent database of the user"
         }
       }
+    },
+    "QueryTable": {
```

Comment:
Maybe it goes by a different name but I think we already have this object somewhere.  Most of the GetUserTables should return an ITable object

Reply:
Found it. It was in TasksCommon.json.